bindata/etcd/cluster-backup.sh

```diff
@@ -28,18 +28,40 @@ if [ ! -d "$1" ]; then
   mkdir -p "$1"
 fi
 
+function check_if_operator_is_progressing {
+   operator="$1"
+
+   export KUBECONFIG="/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig"
+   if [ ! -f "${KUBECONFIG}" ]; then
+      echo "Valid kubeconfig is not found in kube-apiserver-certs. Exiting!"
+      exit 1
+   fi
+
+   progressing=$(oc get co | awk -v OPERATOR=$operator '$1==OPERATOR { print $4 }')
+   if [ "$progressing" != "False" ]; then
+      echo "Currently the $operator is Progressing. The cluster is not healthy. Aborting!"
```

Comment:
s/The cluster is not healthy/A reliable backup requires that a rollout is not in progress/

Reply:
Done. :+1: